client/src/unifyfs-fixed.c

```diff
@@ -369,7 +369,8 @@ int unifyfs_fid_logio_write(int fid,
                             unifyfs_filemeta_t* meta,
                             off_t pos,
                             const void* buf,
-                            size_t count)
+                            size_t count,
+                            size_t* bytes)
```

Comment:
I would rename `bytes` to `nwritten` so it's clearer what it is.

Reply:
Renamed bytes to nwritten/nread depending on the call.